src/dotnet-interactive-vscode-common/src/extension.ts

```diff
@@ -139,14 +139,30 @@ export async function activate(context: vscode.ExtensionContext) {
                 const requestInput = <contracts.RequestInput>commandInvocation.commandEnvelope.command;
                 const prompt = requestInput.prompt;
                 const password = requestInput.isPassword;
-                const value = await vscode.window.showInputBox({ prompt, password });
-                commandInvocation.context.publish({
-                    eventType: contracts.InputProducedType,
-                    event: {
-                        value
-                    },
-                    command: commandInvocation.commandEnvelope,
-                });
+                const isConnectionString = requestInput.isConnectionString;
+                let value: string | undefined;
+                if (isAzureDataStudio(context) && isConnectionString) {
+                    value = await versionSpecificFunctions.getSqlConnectionString();
+                } else {
+                    value = await vscode.window.showInputBox({ prompt, password });
+                }
+                if (!value) {
+                    commandInvocation.context.publish({
+                        eventType: contracts.ErrorProducedType,
+                        event: {
+                            message: 'No input was provided.'
```

Comment:
localize?

Reply:
I'm not seeing localization for any other strings in the extension.

Reply:
We aren't yet localizable.